view-user/src/components/common/Modal/Modal.module.css

```
@@ -1,28 +1,44 @@
 .wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
   position: fixed;
   top: 0;
   left: 0;
   z-index: 2;
-  display: flex;
-  align-items: center;
-  justify-content: center;
   width: 100%;
   height: 100%;
   contain: content;
 }
 
-.content {
-  position: relative;
-  z-index: 50;
-  box-sizing: border-box;
-  object-fit: cover;
-  padding: 40px;
-  overflow: hidden;
-  backdrop-filter: blur(1rem);
-  border: 1px solid rgba(255, 255, 255, 0.4);
-  border-radius: 5px;
-  animation: zoomIn 0.8s cubic-bezier(0.25, 1, 0.5, 1) 1 forwards;
-  will-change: transform, opacity;
+.content p {
+  font-family: "Noto Sans JP";
+  font-size: 6vw;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  color: var(--Main-Color, #ff3342);
+  text-align: center;
+}
+
```

```suggestion
//多分この辺の行
```
```
.background {
  position: absolute;
  z-index: 25;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: rgba(0 0 0 / 50%);
}
```
backgroundをcontent pの下に移動してほしいです！
みやすさとしてcssをまとめておきたいので！！